examples/RadioButton/e2e/radio_button_example.py

```diff
@@ -0,0 +1,9 @@
+import streamlit as st
+from radio_button import custom_radio_button
+
+result = custom_radio_button(
```

Comment:
Can you remove code from https://github.com/streamlit/component-template/blob/f209c94c1f85a7a659b980abf3b3912c035dadcf/examples/RadioButton/radio_button/__init__.py#L29C1-L34?

Reply:
We decided with Przemek to do that later in another PR. Is that ok for you?

Reply:
OK